framework/audio/iaudiobuffer.h

```diff
@@ -16,39 +16,26 @@
 //  along with this program; if not, write to the Free Software
 //  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 //=============================================================================
-
-#ifndef MU_AUDIO_LOOPSTREAM_H
-#define MU_AUDIO_LOOPSTREAM_H
+#ifndef MU_AUDIO_IAUDIOBUFFER_H
+#define MU_AUDIO_IAUDIOBUFFER_H
 
 #include <memory>
-
 #include "iaudiosource.h"
-#include "audiotypes.h"
 
 namespace mu {
 namespace audio {
```

Comment:
Better to use a new style for nested namespaces:
namespace mu::audio {

Reply:
fixed